web3-dev/decentralized-web.rst

```diff
@@ -6,90 +6,64 @@
+For each of the steps above, you will have the choice to do it manually or automatically via Thyra.
 
-    function createWebsite(): void {
-        const bytes = include_base64('./site.zip');
-        Storage.set_data("massa_web", bytes);
-    }
 
-    export function main(_args: string): i32 {
-        createWebsite();
-        print("Uploaded site");
-        return 0;
-    }
+*What is Thyra*
+===============
 
-This smart-contract and the dependencies are available `here <https://github.com/massalabs/massa-sc-examples/tree/main/website>`_.
+θύρα to pronounce thùra and to write Thyra is way simpler that his name looks like! It means entrance, gateway in ancient greek. Thyra is thus a gateway to Massa blockchain.
 
-Setting the DNS
-===============
+Thyra has been developed by Massa Labs to allow any type of users to participate, to create, and to access the blockchain flawlessly using classic web2 while assuring fully decentralised and secure transactions.
+
+In the context of the decentralised web, Thyra acts as a Top Level Domain (TLD) that is redirecting users to the correct smart contract and address to access a specific website
 
-To claim a DNS address for you smart-contract you can use the following smart-contract:
+In a wider context, Thyra acts as a services enabler that will allow you to connect and perform all sorts of actions on the blockchain easily without cutting security or integrity.
 
-.. code-block:: typescript
 
-    import { call } from "massa-sc-std";
-    import { JSON } from "json-as";
+Install Thyra
+======================
+
 
-    @json
-    export class SetResolverArgs {
-        name: string = "";
-        address: string = "";
-    }
+Automatically
+---------------
 
-    export function main(_args: string): i32 {
-        const sc_address = "2cVNfo79K173ddPwNezMi8WzvapMFojP7H7V4reCU2dk6QTeA"
-        call(sc_address, "setResolver", JSON.stringify<SetResolverArgs>({name: "flappy", address: "Gr2aeZt7ZRb9S5SKgAEV1tZ6ERLHGhBCZsAp2sdB6i3rDK9M7"}), 0);
-        return 0;
-    }
+- [MacOS installation script](https://github.com/massalabs/thyra/blob/255c6136c6910d17cbb94ce06b1e908d8516713a/scripts/macos_install.sh)
+- [Windows installation script](https://github.com/massalabs/thyra/blob/255c6136c6910d17cbb94ce06b1e908d8516713a/scripts/thyra-installer.bat)
 
-For now DNS addresses can only be claimed using the following address: `9mvJfA4761u1qT8QwSWcJ4gTDaFP5iSgjQzKMaqTbrWCFo1QM` (and its associated secret key).
+*NB: if you install Thyra with this script using an Ethernet connection, there is high level of chances that you will need to re-install it if you move to wifi. Sorry about that, we’re working on it!*
 
-Once you've done this step, you should be able to access to your website using
-`Massa's browser plugin <https://github.com/massalabs/massa-wallet>`_ at `https://flappy.massa`.
+- [Linux Ubuntu installation script](https://github.com/massalabs/thyra/blob/main/scripts/linux_install.sh)
```

Comment:
Nothing opens automatically. The way it works now is: 
you run the script using the cmd 
at the end of the install, the script is telling: "successfully installed"
Then you go back to the documentation to follow the next steps, which is: create your wallet.

That was my thinking process. So I don't ask user to open anything yet, because they'll do it right after. 
Is that clear enough?